api/notd/model.py

```diff
@@ -23,6 +23,23 @@ class RetrievedTokenTransfer:
     blockHash: str
     blockDate: datetime.datetime
 
+@dataclasses.dataclass(frozen=True, unsafe_hash=True)
+class ERC1155TokenTransfer:
+    transactionHash: str
+    registryAddress: str
+    operatorAddress: str
+    fromAddress: str
+    toAddress: str
+    tokenId: Optional[str]
```

Comment:
these three shouldnt be optional

Reply:
was testing something sorry

Reply:
ah no worries, you can ignore my comments if its still wip 👍